low-level/sspl_ll_d

```diff
@@ -780,8 +780,8 @@ if __name__ == "__main__":
     # Initialize logging
     try:
         logging_level = Conf.get(SSPL_CONF, f"{SYSTEM_INFORMATION}>{LOG_LEVEL}", "INFO")
-        init_logging("SSPL-LL", logging_level, SYSLOG_HOST, SYSLOG_PORT)
-
+        logger.init(service_name="sspl", log_path=SSPL_LOG_PATH, level=logging_level)
+        logger.info("Logging has been initialized for sspl")
```

Comment:
keep SSPL is capital. atleast in new / touched code we can maintain that

Reply:
okay